### PR TITLE
operators

### PR DESCRIPTION
**Pull request by untzag**
- [x] `__iadd__`
- [x] `__imul__`
- [x] `__ipow__`
- [x] `__isub__`
- [x] `__itruediv__`
- [x] `symmetric_root`
- [x] remove `channel.invert`
- [x] `clip`
- [x] `log`, `log10`, `log2`
- [x] `channel.minor_extent`, `channel.major_extent`, `channel.mag`
- [x] `channel.normalize`

### CODE REVIEW

File: WrightTools/_dataset.py [hunk 1]
```diff
@@ -25,8 +27,53 @@ class Dataset(h5py.Dataset):
...
+    def __isub__(self, value):
+        def f(dataset, s, value):
+            if hasattr(value, 'shape'):
+                dataset[s] -= value[wt_kit.valid_index(s, value.shape)]
+            else:
+                dataset[s] -= value
+        self.chunkwise(f, value=value)
+        return self
+
+    def __itruediv__(self, value):
```
Comment (ksunden) [inline WrightTools/_dataset.py]: Should we add `__ifloordiv__` and `__imod__` for completeness sake

File: WrightTools/data/_channel.py [hunk 2]
```diff
@@ -113,59 +104,17 @@ def signed(self):
...
-    def invert(self):
-        """Invert channel values."""
-        self[:] *= -1
-
     def mag(self):
         """Channel magnitude (maximum deviation from null)."""
         return self.major_extent
 
-    def normalize(self, axis=None):
+    def normalize(self):
```
Comment (ksunden) [inline WrightTools/data/_channel.py]: What if we wish to normalize along a particular axis?

File: tests/data/channel/normalize.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+def test_wigner():
+    p = datasets.COLORS.v2p2_WL_wigner
+    data = wt.data.from_COLORS(p)
+    data.ai0.normalize()
+    assert data.ai0.null == 0.
+    assert data.ai0.max() == 1.
+    data.close()
+
+
```
Comment (ksunden) [inline tests/data/channel/normalize.py]: Test for signed data normalization?

File: WrightTools/data/_channel.py [hunk 4]
```diff
@@ -91,17 +93,6 @@ def null(self):
             self.attrs['null'] = 0
         return self.attrs['null']
 
-    @property
-    def info(self):
```
Comment (ksunden) [inline WrightTools/data/_channel.py]: Any particular reason to remove the info dictionary?

File: WrightTools/_dataset.py (hunk 1 above)
Comment (untzag) [reply WrightTools/_dataset.py]: my vote is to add them later, only if we can think of some good reason for their existance

File: WrightTools/data/_channel.py (hunk 4 above)
Comment (untzag) [reply WrightTools/data/_channel.py]: yeah, I just wasn't seeing it get used

do you think it's useful?

File: WrightTools/data/_channel.py (hunk 2 above)
Comment (untzag) [reply WrightTools/data/_channel.py]: yeah, I was disapointed to do this, but I think it just needs to be thought about

I'll open an issue

File: tests/data/channel/normalize.py (hunk 3 above)
Comment (untzag) [reply tests/data/channel/normalize.py]: yeah OK

believe it or not, I don't think I have any signed datasets in WrightTools

File: WrightTools/data/_channel.py (hunk 2 above)
Comment (ksunden) [reply WrightTools/data/_channel.py]: Perhaps add it to #309

File: tests/data/channel/normalize.py (hunk 3 above)
Comment (untzag) [reply tests/data/channel/normalize.py]: actually, I'm not sure what behavior I want for signed datasets

**Pull request merged**